annotator/dirstral_annotator/recognizers/base.py

```
@@ -0,0 +1,62 @@
+        raise ValueError("fps must be positive")
+    with tempfile.TemporaryDirectory(prefix="dirstral-frames-") as tmp:
+        out = Path(tmp) / "frame-%08d.jpg"
+        cmd = [
+            ffmpeg, "-hide_banner", "-loglevel", "error",
+            "-i", str(media_path),
+            "-vf", f"fps={fps}",
+            "-q:v", "3",
+            str(out),
+        ]
+        try:
+            subprocess.run(cmd, check=True, capture_output=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)